runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -78,18 +79,25 @@ dependencies {
   compile library.java.google_api_client
   compile library.java.google_api_services_clouddebugger
   compile library.java.google_api_services_dataflow
-  compile library.java.google_api_services_storage
+  testImplementation 'com.google.apis:google-api-services-storage:v1-rev20200611-1.30.10'
   compile library.java.google_auth_library_credentials
   compile library.java.google_auth_library_oauth2_http
   compile library.java.google_http_client
-  compile library.java.google_http_client_jackson2
+  testImplementation 'com.google.http-client:google-http-client-jackson2:1.34.0'
```

Comment:
This should be `library.java.google_http_client_jackson2`

For each dependency, please find it in this list: https://github.com/apache/beam/blob/b131cc88c85cde646514ba7f0385e1b5f768ae0d/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy#L463